sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -829,9 +790,46 @@ pi_result _pi_device::initialize(int SubSubDeviceOrdinal,
         ZE_CALL_NOCHECK(zeDeviceGetCacheProperties,
                         (ZeDevice, &Count, &Properties));
       };
+
+  // Check device id for PVC.
+  // TODO: change mechanism for detecting PVC once L0 provides an interface.
+  ImmCommandListsPreferred = (ZeDeviceProperties->deviceId & 0xff0) == 0xbd0;
```

Comment:
Did we request a Level-Zero API to tell when immediate submission is preferable for the device?

Reply:
Yes, but the general feeling is that immediate commandlists will be the default everywhere. For now, they are known to work best on PVC. On other GPUs they may or may not work. Once known to work correctly on all GPUs we will remove this limited use of immediate commandlists.